utils/zerovec/src/varzerovec/borrowed.rs

```diff
@@ -98,20 +98,15 @@ impl<'a, T: VarULE + ?Sized> VarZeroVecBorrowed<'a, T> {
                 marker: PhantomData,
             });
         }
-        let len_bytes = slice.get(0..4).ok_or(VarZeroVecError::FormatError)?;
-        let len_ule = PlainOldULE::<4>::parse_byte_slice(len_bytes)
-            .map_err(|_| VarZeroVecError::FormatError)?;
+        let len_bytes = slice.get(0..4).ok_or(ULEError::FormatError)?;
+        let len_ule =
+            PlainOldULE::<4>::parse_byte_slice(len_bytes).map_err(|_| ULEError::FormatError)?;
```

Comment:
Thought: Should we add special parsing methods to `PlainOldULE` that are infallible?

Reply:
We already have `From<[u8; N]>`, but it's pretty moot here since it's tricky to statically prove that a subslice is of a given length (until someone writes a `slice<N1, N2>` function)